src/quo2/core.cljs

```diff
@@ -91,12 +91,14 @@
     quo2.components.profile.profile-card.view
     quo2.components.profile.select-profile.view
     quo2.components.profile.showcase-nav.view
-    quo2.components.reactions.reaction
     quo2.components.record-audio.record-audio.view
     quo2.components.record-audio.soundtrack.view
     quo2.components.selectors.disclaimer.view
     quo2.components.selectors.filter.view
-    quo2.components.selectors.reactions.view
+    quo2.components.selectors.reactions-selector.view
+    quo2.components.selectors.react.view
+    quo2.components.selectors.react-selector.view
+    quo2.components.selectors.react-selector-add.view
```

Comment:
what is react-selector-add? 🤔 
From what I see this is just a variant of React-Selector and ideally we just have it that so we map 1-1 with figma

Reply:
it's this right? - https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=3158-50223&mode=design&t=qXO1XqJBWLT0LfJa-4

Reply:
I based this on the [reply](https://github.com/status-im/status-mobile/issues/17245#issuecomment-1713840957) from @Francesca-G. We could make `add` to be another state for the component that is activated by a prop `add?`, but seemed like the uglier choice even if consistent with the design system.

Reply:
Yeah it's perhaps the uglier choice but we have made a decision in the project to keep the components names aligned with the design system and the api matching figma where possible - this makes it easy to track and maintain the component library then as it is a self documenting process.

Reply:
i.e this should be configured by `[quo/react-selector {:state :add-reaction}]`
<img width="363" alt="Screenshot 2023-09-18 at 10 40 07" src="https://github.com/status-im/status-mobile/assets/22799766/b1bc2eca-0ca2-4afa-b889-d7afd1cc4e12">

Reply:
I understand refactoring all of this to align might be too much at one time but I think the first priority of this work should be to get the file system/ quo.core aligned correctly with Figma

Reply:
Yep, will take that into account for the refactoring as well.